install/updates/froxlor/0.10/update_0.10.inc.php

```diff
@@ -649,3 +649,20 @@
 	showUpdateStep("Updating from 0.10.18 to 0.10.19", false);
 	\Froxlor\Froxlor::updateToVersion('0.10.19');
 }
+
+if (\Froxlor\Froxlor::isDatabaseVersion('202005150')) {
+	showUpdateStep("Add new performerce indexes", false);
```

Comment:
second parameter needs to be `true` if you intend to add a status afterwards (using `lastStepStatus`). Parameter `false` indicates that this step has no (closing) status-text.

And a little typo: it's **performance** and not _performerce_

Reply:
Ok. I fix this.
Sorry for my errors